sdks/python/apache_beam/pipeline_test.py

```diff
@@ -185,65 +180,6 @@ def apply(self, pcoll):
         ['a-x', 'b-x', 'c-x'],
         sorted(['a', 'b', 'c'] | 'AddSuffix' >> AddSuffix('-x')))
 
-  def test_cached_pvalues_are_refcounted(self):
```

Comment:
Done. Added the test. It uses unix only resource module and skipped on other platforms.